cloud-aws-cloudformation/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/AwsResourceService.java

```diff
@@ -0,0 +1,54 @@
+package com.sequenceiq.cloudbreak.cloud.aws.connector.resource;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import com.amazonaws.services.ec2.model.DescribeInstanceTypesRequest;
```

Comment:
It looks like all of Cloudbreak uses the "com.amazonaws.services.ec2" packages to talk to AWS ... and not the one being used here. I don't think we should change that.

Reply:
all of cloudbreak uses software.amazon.awssdk.services.ec2.model - that is the same package I am using here.

Reply:
Correction to my original comment. All of cloudbreak uses "software.amazon.awssdk.*". What is being used here is "com.amazonaws."

Reply:
changed it - that was changed in the master, and I pulled it this morning and updated the packages